### PR TITLE
Remove the body initialization for null.

### PR DESCRIPTION
**Pull request by raych1**
Body is not a required property and can be undefined.

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: runtime/ms-rest/lib/webResource.js [hunk 1]
```diff
@@ -25,7 +25,6 @@ class WebResource {
     this.url = null;
     this.method = null;
     this.headers = {};
-    this.body = null;
```
Comment (amarzavery) [inline runtime/ms-rest/lib/webResource.js]: Instead of removing it. Please set the body to `undefined`.

**Comment by amarzavery** [comment]
Can you replace the double quote with single quote as mentioned in the CI over [here](https://travis-ci.org/Azure/azure-sdk-for-node/jobs/551604845#L1361).

**Comment by amarzavery** [comment]
Please update the [package.json](https://github.com/Azure/azure-sdk-for-node/blob/master/runtime/ms-rest/package.json) version to `2.5.2` and update the [Changelog.md](https://github.com/Azure/azure-sdk-for-node/blob/master/runtime/ms-rest/Changelog.md)

File: runtime/ms-rest/lib/webResource.js (hunk 1 above)
Comment (raych1) [reply runtime/ms-rest/lib/webResource.js]: Updated.

**Comment by raych1** [comment]
> fix as per comments.

@amarzavery, fix was submitted, pls review.

**Pull request merged**